sdk/KoreBuild/modules/sharedsources/sharedsources.csproj

```diff
@@ -27,11 +36,11 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(NuspecBasePath)'!=''">
-    <Compile Include="$(NuspecBasePath)\**\*.cs">
+    <Compile Include="$(NuspecBasePath)\**\*.cs" Exclude="$(BaseOutputPath);$(BaseIntermediateOutputPath);">
```

Comment:
If these `Exclude`s showed up a third time I'd say they should be a property like "NonCompilingPaths" for descriptiveness and DRY, but with just two instances it's more of a preference.